### PR TITLE
feat: add support for rememberToken() shorthand

### LINKED ISSUE #228: rememberToken is generating a empty faker
- Laravel Version: 7.12.0
- PHP Version: 7.4.6
- Blueprint Version: 1.12.0
- Platform: Windows | Linux (WSL)

### Description:
I have a remember_token column in the User-migration.
When generating that one, it generates an empty $faker-> .
Would it be possible to set it to the default 'remember_token' => Str::random(10), or maybe $faker->asciify(10) ?

#### One more small request (not that it's really needed):
Would it be possible to just write remember_token without the column name to generate $table->rememberToken(); like in the original migration?

### Steps To Reproduce:
Run the buildprint:build with my draft.yaml :)

### draft.yaml:
```yaml
models:
  User:
    name: string
    email: string unique
    email_verified_at: datetime nullable
    password: string
    remember_token: remembertoken
```
### Suggested draft.yaml for the small request
```yaml
models:
  User:
    name: string
    email: string unique
    email_verified_at: datetime nullable
    password: string
    remember_token:
```

### PR DESCRIPTION
**Pull request by nathane**
- Add the shorthand to `Blueprint::parse`
- Update the `MigrationGenerator` for `rememberToken`
- Update the `FactoryGenerator` for `remember_token` columns
- Update the `ModelGenerator` to hide the attribute

Closes #228

### CODE REVIEW

**Comment by jasonmccreary** [comment]
Dude, what is your Twitter handle so I can give you some praise.

**Comment by nathane** [comment]
it's `@nathanesayeas`.

File: tests/Feature/Generator/FactoryGeneratorTest.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function output_writes_nothing_for_empty_tree()
      * @test
      * @dataProvider modelTreeDataProvider
      */
-    public function output_writes_migration_for_model_tree($definition, $path, $migration)
+    public function output_writes_factory_for_model_tree($definition, $path, $factory)
```
Comment (jasonmccreary) [inline tests/Feature/Generator/FactoryGeneratorTest.php:51]: Thanks for scouting this.

**Comment by jasonmccreary** [comment]
Looks like one of the tests is failing due to the ordering of imports. This is my doing. But I can't edit it on the web.

So I'm going to merge and fix the failure.

**Pull request merged**